features/records/components/ShowRecord.tsx

```diff
@@ -101,7 +106,20 @@ const ShowRecord = () => {
           <title>View record {data?.data?.id} | 👋 Hi!</title>
         </Head>
         {isLoading && <LoadingOverlay transparent={isEmpty(data?.data)} />}
-        {error && <div>Error: {JSON.stringify(error)}</div>}
+        {error && (
+          <ErrorMessage
+            message={
+              "status" in error && error?.status === 404
+                ? "404, Record not found"
+                : undefined
+            }
+            error={
+              "status" in error && error?.status === 404
+                ? ""
+                : JSON.stringify(error)
+            }
+          />
+        )}
```

Comment:
Can we refactor this to be a bit easier to read?

```suggestion
        {error && "status" in error && error?.status === 404 && (
          <ErrorMessage
            message="404, Record not found"
            error={JSON.stringify(error)}
          />
        )}
```